packages/patternfly-4/react-core/src/layouts/Page/Page.js

```diff
@@ -72,16 +73,18 @@ class Page extends React.Component {
       } else {
         this.mainRef.current.removeEventListener('scroll', this.handleScroll);
       }
-      window.removeEventListener('resize', this.handleResize);
     }
+    window.removeEventListener('resize', this.handleResize);
```

Comment:
You might check out  match media and addListener it will allow using the same query as core and is less exepensive than resize. https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia